base/tools/src/test/shell/test_PKICertImport.bash

```diff
@@ -470,6 +550,7 @@ function test_leaf_fail_no_root() {
 
 function main() {
     time (
+	generate_certificates || exit $Z
         test_cert_import_root || exit $?
         test_cert_import_server || exit $?
         test_cert_missing_intermediate || exit $?
```

Comment:
Could you adjust the indentation too?

Reply:
This is strange. If I open with the my editors (`vim` and `emacs`) is correct, spaces all lines. If I go to file changed is not correct but if I click on view the file is correct.

I think it is a visualisation  problem!

Reply:
At the end I change a different line and it is updated.